yash-datalake/nested/dataFactory.json

```diff
@@ -0,0 +1,316 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "KibanaESDeployedVMIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the data factory. Must be globally unique."
+      }
+    },
+    "dataFactoryName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the data factory. Must be globally unique."
+      }
+    },
+	"_artifactsLocation": {
+          "type": "string",
+          "metadata": {
+              "description": "The base URI where artifacts required by this template are located including a trailing '/'"
+          }
+      },	  
+    "_artifactsLocationSasToken": {
+          "type": "securestring",
+          "metadata": {
+              "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated. Use the defaultValue if the staging location is not secured."
+          }
+      },
+    "dataFactoryLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Location of the data factory. Currently, only East US and East US 2 are supported. "
+      }
+    },
+    "blobContainer": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the blob container in the Azure Storage account."
+      }
+    },
+    "clientAzureStorageConnectionString": {
+      "type": "string",
+      "metadata": {
+        "description": "Connection string for the Azure Storage account."
+      }
+    },
+    "servicePrincipalId": {
+      "type": "string",
+      "metadata": {
+        "description": "The ID of the service principal that has permissions to create HDInsight clusters in your subscription."
+      }
+    },
+    "servicePrincipalKey": {
+      "type": "string",
+      "metadata": {
+        "description": "The access key of the service principal that has permissions to create HDInsight clusters in your subscription."
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "storage account name"
+      }
+    }
+  },
+  "variables": {
+    "yashAzureStorageLinkedServiceName": "LinkedServiceToYashAzureStorage",
+    "SASUri": "https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl",

```

Comment:
what's the requirement for this?  We can't have it pointing to a private storage account for CI/CD

Reply:
The private storage account for CI/CD contains datasets that are being used in this Quickstart portal.There are 4 datasets of upto 500mb each.

Reply:
we'll need to move them to storage used by the pipeline - you can send me a sasuri for a container or each and I'll copy them over...

Reply:
Use below URL's to copy dataset files used in our solution:

1. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/demographics/demographics_data_20170520.csv?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl
2. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/orders/order_data.csv?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl
3. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/customers/customerData.csv?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl
4. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/products/products.csv2013?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl
5. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/products/products.csv2014?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl
6. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/products/products.csv2015?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl
7. https://yashinnovationstorage.blob.core.windows.net/yash-datalake-resource/datasets/products/products.csv2016?sv=2017-11-09&sr=c&sig=qoJfA%2BX7K7YdOAjqZFFfuv4C8MVbbQkeBhugnG7DoKk%3D&se=2023-06-20T10%3A55%3A45Z&sp=rl

Reply:
I'm getting a 404